command/operator_debug_test.go

```diff
@@ -578,6 +578,36 @@ func TestDebug_Fail_Pprof(t *testing.T) {
 	require.Contains(t, ui.OutputWriter.String(), "Created debug archive")   // Archive should be generated anyway
 }
 
+// TestDebug_PprofVersionCheck asserts that only versions 0.11.0 -> 0.11.2 match
+// the version constraint.
+func TestDebug_PprofVersionCheck(t *testing.T) {
```

Comment:
Since retrieving the version number from a specific Nomad agent was not straightforward, I started with the test and worked my way backwards through implementation.